EIPS/eip-5058.md

```diff
@@ -0,0 +1,210 @@
+---
+eip: 5058
+title: Lockable ERC-721 Standard
+description: Lockable ERC-721 tokens
```

Comment:
```suggestion
description: Lockable EIP-721 and EIP-1155 Tokens
```

Reply:
The protocol is currently only implemented for eip-721.